UI/libman.json

```diff
@@ -10,6 +10,33 @@
         "dist/browser/signalr.js",
         "dist/browser/signalr.min.js"
       ]
+    },
```

Comment:
this part is generated by VS, can you explain what you did these changes?

Reply:
> this part is generated by VS, can you explain why you did these changes?

Since adding signalR refres by default path suggested by vs microsoft/dis/XXX hence i considered the default path we can change to any path we required that wont be an issue

Reply:
See ,y commit, I left only actual path